cmip6_downscaling/workflows/common_tasks.py

```diff
@@ -0,0 +1,271 @@
+import os
+
+os.environ['PREFECT__FLOWS__CHECKPOINTING'] = 'true'
+
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import xarray as xr
+from prefect import task
+from skdownscale.pointwise_models.utils import default_none_kwargs
+from xpersist.prefect.result import XpersistResult
+
+from cmip6_downscaling.config.config import CONNECTION_STRING, intermediate_cache_store, serializer
+from cmip6_downscaling.data.cmip import get_gcm, get_gcm_grid_spec, load_cmip
+from cmip6_downscaling.data.observations import get_obs
+from cmip6_downscaling.methods.bias_correction import (
+    bias_correct_gcm_by_method,
+    bias_correct_obs_by_method,
+)
+from cmip6_downscaling.workflows.paths import (
+    build_gcm_identifier,
+    build_obs_identifier,
+    make_bias_corrected_gcm_path,
+    make_bias_corrected_obs_path,
+    make_coarse_obs_path,
+    make_interpolated_gcm_path,
+    make_interpolated_obs_path,
+)
+from cmip6_downscaling.workflows.utils import rechunk_zarr_array_with_caching, regrid_ds
+
+
+@task
```

Comment:
General comment for the tasks here. Do you have any thoughts on if we should define the functions with the task decorator (and all the caching stuff) here, or in the flow. I still don't know which is better, but it would be nice to stick to one way.

Reply:
hmm yea i don't have too much preference. i was sticking to the unspoken rule of only having tasks in the `workflows` (and now `tasks`) sub dir, but avoid having to repeat the task declaration in each workflow if possible. thoughts?